yarn.lock

```diff
@@ -2005,6 +2005,17 @@
     "@react-stately/toggle" "^3.2.3"
     "@react-types/button" "^3.4.1"
 
+"@react-aria/focus@^3.10.0":
```

Comment:
looks like there's some duplicates of things
might need to do 
```
npx yarn-deduplicate
yarn install
```

Reply:
last type I ran this command, I got an alert saying that this is now done automatically 🤷‍♀️